nsot/api/views.py

```diff
@@ -662,33 +662,68 @@ def get_serializer_class(self):
             return serializers.InterfaceUpdateSerializer
         if self.request.method == 'PATCH':
             return serializers.InterfacePartialUpdateSerializer
-
         return self.serializer_class
 
     @detail_route(methods=['get'])
     def addresses(self, request, pk=None, site_pk=None, *args, **kwargs):
         """Return a list of addresses for this Interface."""
         interface = self.get_resource_object(pk, site_pk)
         addresses = interface.addresses.all()
-
         return self.list(request, queryset=addresses, *args, **kwargs)
 
     @detail_route(methods=['get'])
     def assignments(self, request, pk=None, site_pk=None, *args, **kwargs):
         """Return a list of information about my assigned addresses."""
         interface = self.get_resource_object(pk, site_pk)
         assignments = interface.assignments.all()
-
         return self.list(request, queryset=assignments, *args, **kwargs)
 
     @detail_route(methods=['get'])
     def networks(self, request, pk=None, site_pk=None, *args, **kwargs):
         """Return all the containing Networks for my assigned addresses."""
         interface = self.get_resource_object(pk, site_pk)
-
         return self.list(request, queryset=interface.networks, *args, **kwargs)
 
     @detail_route(methods=['get'])
+    def parent(self, request, pk=None, site_pk=None, *args, **kwargs):
+        """Return parent of interface"""
```

Comment:
Can you please update the docstring to be similar in style to the one at `nsot.api.NetworkViewSet.parent()`?

And for the remaining methods you've added can you please make sure they all have docstrings?

This is important because the docstrings are used in the automated API documentation on nsot.readthedocs.io and in the browsable API.